examples/js_of_ocaml/.jshintignore

```diff
@@ -0,0 +1 @@
+js/todomvc.ml
```

Comment:
Isn't there a better way to ignore files then per-example `jshintignore` files? @samccone any ideaz?